### PR TITLE
Stubbing out the doc structure for several new models.

### PR DESCRIPTION
**Pull request by ksbeattie**
This is a new outline structure for the documentation for several models.

The details of the models themselves still need to be added.

### CODE REVIEW

File: docs/models/fossil/ccs.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
+CCS
+===
+
+The DISPATCHES CCS Model
+
+Degrees of Freedom
+------------------
+
+The CCS Model as X degrees of freedom.
```
Comment (dangunter) [inline docs/models/fossil/ccs.rst:9]: typo "as" => "has"

File: docs/models/fossil/concrete_thermal_storage.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
+Concrete Thermal Storage
+========================
+
+The DISPATCHES Concrete Thermal Storage Model
+
+Degrees of Freedom
+------------------
+
+The Concrete Thermal Storage Model as X degrees of freedom.
```
Comment (dangunter) [inline docs/models/fossil/concrete_thermal_storage.rst:9]: typo "as" => "has"

File: docs/models/nuclear/electrolizer.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
+Electrolizer
```
Comment (dangunter) [inline docs/models/nuclear/electrolizer.rst:1]: "Electrolyzer"

File: docs/index.rst [hunk 4]
```diff
@@ -1,14 +1,69 @@
...
-=====================================
+DISPATCHES
+==========
+The **D**\ esign **I**\ ntegration and **S**\ ynthesis **P**\ latform to **A**\ dvance **T**\ ightly
+**C**\ oupled **H**\ ybrid **E**\ nergy **S**\ ystems (DISPATCHES) project, is being developed and
+used to identify and optimize Integrated Energy Systems for operation within the bulk power system
+via energy market signals.
 
+DISPATCHES is part of the DOE Grid Modernization Laboratory Consortium (GMLC).
+
```
Comment (dangunter) [inline docs/index.rst:12]: I think you should add links to the github repo and home page.

**Review by dangunter — DISMISSED** [review]
Noted some typos and places where links could be added, but looks OK.

File: docs/index.rst (hunk 4 above)
Comment (ksbeattie) [reply docs/index.rst:13]: I added a link for DISPATCHES to the github repo, but what home page do you mean?

**Review by andrewlee94 — APPROVED** [review]
A couple of minor comments.

File: docs/index.rst (hunk 4 above)
Comment (ksbeattie) [reply docs/index.rst:14]: ok, I linked to the github.io site and the GMLC main page here.

File: docs/models/fossil/ultra_super_critical.rst [hunk 5]
```diff
@@ -0,0 +1,37 @@
+Ultra Super Ctitical
```
[the remark this answers is not in the document]
Comment (ksbeattie) [reply docs/models/fossil/ultra_super_critical.rst]: ok, changes made and renamed the file itself to "ustg"

File: docs/models/nuclear/electrolyzer.rst [hunk 6]
```diff
@@ -0,0 +1,37 @@
+Electrolyzer
```
[the remark this answers is not in the document]
Comment (ksbeattie) [reply docs/models/nuclear/electrolyzer.rst]: done

**Review by dangunter — APPROVED** [review]
LGTM

**Review by andrewlee94 — APPROVED** [review]

**Pull request merged**